### ISSUE TITLE
Discuss data (and model) update cycle for operational deployment

### ISSUE THREAD

**Issue by robknapen** [soilwise-he/natural-language-querying#10]
New data will be harvested periodically which will somehow be reflected in the NLQ component. E.g. it might need to updating of embeddings in a vector store.

New versions of LLMs are made available frequently at the moment, updating of what is used by the NLQ component might also be needed accordingly. I.e. for security or model toxicity reasons.

**Comment by BerkvensNick** [comment]
Good point, I think this can only be discussed when we have defined a more clear use case we want to develop/set up
